app/static/src/app/components/WodinPlot.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="wodin-plot-container" :style="plotStyle">
+    <div class="plot" ref="plot">
+    </div>
+    <div v-if="!hasPlotData" class="plot-placeholder">
+      {{ placeholderMessage }}
+    </div>
+    <slot></slot>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+    computed, defineComponent, ref, watch, onMounted, onUnmounted, PropType
+} from "vue";
+import { EventEmitter } from "events";
+import {
+    newPlot, react, PlotRelayoutEvent, Plots
+} from "plotly.js";
+import { WodinPlotData } from "../plot";
+import { OdinSolution } from "../types/responseTypes";
+
+export default defineComponent({
+    name: "WodinPlot",
```

Comment:
I suggest `WodinPlotODE` perhaps here, as this contains logic that is fairly specific to coping with the solutions to ODEs where we can rescale the final thing. For the other sensitivity plot types, and for the stochastic plots where we just have the data, we won't have any of the nastier logic here